xn2v/text_encoder_tf.py

```diff
@@ -171,9 +171,13 @@ def build_dataset(self, max_vocab=50000) -> Tuple[Union[tf.data.Dataset, tf.Ragg
         if max_vocab != len(count_list):
             raise ValueError('The length of count_as_tuples does not match max_vocab_size.')
         else:
-            try:
-                tensor_data = tf.data.Dataset.from_tensor_slices(sequences)
-            except ValueError:
-                tensor_data = tf.ragged.constant(sequences)  # for nested lists of differing lengths
+            #try:
+            #    tensor_data = tf.data.Dataset.from_tensor_slices(sequences)
+            #except ValueError:
+            #    tensor_data = tf.ragged.constant(sequences)  # for nested lists of differing lengths
```

Comment:
This comment points to the one above, do we know that the changes made to the rest of the code work for both of these tf objects?